ui/src/message_popup/instructions_card.jsx

```diff
@@ -26,50 +26,39 @@ export default React.createClass({
   getInitialState(){
     return ({
       isSolutionMode: _.has(this.props.itemsToShow, "solution"),
-      email: this.props.email,
-      sessionLength: this.props.sessionLength,
-      questions: allQuestions,
       competencyGroupValue: ALL_COMPETENCY_GROUPS,
-      shouldShowStudentCard: true,
-      shouldShowSummary: true,
-      helpType: this.props.helpType
+      scaffolding: { 
+        email: this.props.email,
+        sessionLength: this.props.sessionLength,
+        questions: allQuestions,
+        shouldShowStudentCard: true,
+        shouldShowSummary: true,
+        helpType: this.props.helpType
+      }
```

Comment:
The way the state and props flow now, this component has to have most of the same bits as the experience component above.    That was my original idea, but perhaps in the actual execution it's more complicated so let's try sketching it out and see if there's a way to simplify.  I can swing by later this afternoon or we could do it in the morning tomorrow.